examples/CMakeLists.txt

```diff
@@ -7,10 +7,10 @@ cmake_minimum_required(VERSION 2.8.10)
 #version variable is defined from project root's CMakeLists
 if("${OpenCV_VERSION_MAJOR}$" EQUAL 2)
   message(STATUS "OpenCV 2.x detected")
-  find_package(OpenCV 2.4.3 REQUIRED core imgproc highgui)
```

Comment:
Do you think we can leave 2.4.3 here, or does that cause problems as well? My thought is that if we just put "2", then CMake will not fail if someone tries to run it with OpenCV 2.3, 2.2 or even older, but eos will most likely not work with such an old version.